velox/common/base/tests/GTestUtils.h

```diff
@@ -54,6 +54,13 @@
   VELOX_ASSERT_THROW_IMPL(                                     \
       facebook::velox::VeloxRuntimeError, _expression, _errorMessage)
 
+#define VELOX_ASSERT_ERROR_STATUS(_expression, _statusCode, _errorMessage) \
+  const auto status = (_expression);                                       \
+  ASSERT_TRUE(status.code() == _statusCode);                               \
```

Comment:
Would it be possible to include mismatches codes in the error message? What does the failure look like now?

Reply:
Added code in the error message. Now the message looks like below.
Code mismatch:
```
Expected error code to be 'Type error', but received 'User error'.
```
Message mismatch:
```
Value of: status.message().find(expectedErrorMessage) != std::string::npos
  Actual: false
Expected: true
Expected error message to contain 'Unknown error.', but received 'Result overflows.'.
```

Reply:
Nice. Thank you.